packages/common/test/core/EntityEditor.test.ts

```diff
@@ -95,7 +95,7 @@ describe("EntityEditor:", () => {
       expect(toEditorSpy).toHaveBeenCalled();
       expect(chk.id).toBe("00c");
       await editor.save(chk);
-      expect(fromEditorSpy).toHaveBeenCalledWith(chk);
+      expect(fromEditorSpy).toHaveBeenCalledWith(chk, undefined);
     });
   });
   describe("supports pages:", () => {
```

Comment:
editorContext is optional -- and we don't use it anywhere else yet -- so it's undefined right now